andela/pippin-ah-backend
# #163019081 Configure project to use es6 or later version

#### What does this PR do?
Configure project to use es6 or later version.

#### Description of Task to be completed?
- Install <code>babel dependencies </code> for transpiling.
- Create <code>.babelrc</code> in the root of the project.
- Add <code>babel preset</code> to <code>.babelrc</code> file.

#### How should this be manually tested?
- Use <code>es6</code> features like <code>import</code> instead of <code>require</code> in any file and verify <code>server</code> runs successfully.

### What are the relevant pivotal tracker stories?
#163019081

### Screenshots (if appropriate)
None

#### Any background context you want to provide?
None
